dev/breeze/src/airflow_breeze/utils/run_tests.py

```diff
@@ -327,12 +328,16 @@ def generate_args_for_pytest(
             helm_test_package=helm_test_package,
             python_version=python_version,
         )
+
+    max_fail = 50
+    if database_isolation:
+        max_fail = 1000
```

Comment:
I think at least should be reverted before merge.

Reply:
Or fixed to skip the core parts (that's what I plan to do) - but yes - it's temporary